version.tf

```diff
@@ -5,6 +5,7 @@
  */
 
 terraform {
+  experiments = [module_variable_optional_attrs]
```

Comment:
What does this line do?

Reply:
enables the `optional` modifier. This has been added to 0.15 as experiment and made to 1.3 as stable. But is completely fine and safe to use it.